templates/_workshopeditheader.html

```diff
@@ -0,0 +1,13 @@
+{% if workshop.status == 'Z' %}
```

Comment:
Use Enums instead of magic constants.

Reply:
This is how it works everywhere in the project - add it to 'refactor all the things' list

Reply:
Ok